packages/packages/commonwell-sdk/src/models/commonwell.ts

```diff
@@ -0,0 +1,125 @@
+import * as fs from "fs";
+import { RequestMetadata } from "../client/commonwell";
+import { CertificateParam, CertificateResp } from "./certificates";
+import { DocumentQueryResponse } from "./document";
+import { Identifier, StrongId } from "./identifier";
+import { NetworkLink, PatientLinkProxy } from "./link";
+import { Organization, OrganizationList } from "./organization";
+import { Patient, PatientNetworkLinkResp, PatientSearchResp, PatientLinkResp } from "./patient";
+import { PatientLink, PatientLinkSearchResp, Person, PersonSearchResp } from "./person";
+
+export interface CommonWellType {
+  get oid(): string;
```

Comment:
I think this is only used internally, so in theory no need to be exposed in the interface.